tools/v8-prof/mac-tick-processor

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# A wrapper script to call 'linux-tick-processor' with Mac-specific settings.
+
+tools_path=`cd $(dirname "$0");pwd`
+v8_tools="$tools_path/../../deps/v8/tools"
+$tools_path/linux-tick-processor --mac --nm=$v8_tools/mac-nm $@
```

Comment:
General observation (i.e., applicable to more than just this line): to avoid issues with paths containing spaces, this should presumably be written as:

```sh
"$tools_path/linux-tick-processor" --mac --nm="$v8_tools/mac-nm" $@
```